packages/flare/python/cron_job_ingest_events.py

```diff
@@ -29,46 +32,119 @@ def main():
         raise Exception(str(e))
 
     app: client.Application = splunk_service.apps[APP_NAME]
-    flare_api = FlareAPI(app=app)
+    api_key: Optional[str] = None
+    tenant_id: Optional[int] = None
+    for item in app.service.storage_passwords.list():
+        if item.content.username == "api_key":
+            api_key = item.clear_password
 
-    from_ = get_from_value(app=app)
-    event_feed = flare_api.retrieve_feed(from_=from_)
-    set_from_value(app=app, next=event_feed["next"])
+        if item.content.username == "tenant_id":
+            tenant_id = (
+                int(item.clear_password) if item.clear_password is not None else None
+            )
 
-    if event_feed["items"]:
-        for item in event_feed["items"]:
-            print(json.dumps(item))
+    if not api_key:
+        raise Exception("API key not found")
 
+    if not tenant_id:
+        raise Exception("Tenant ID not found")
+    flare_api = FlareAPI(app=app, api_key=api_key, tenant_id=tenant_id)
 
-def get_from_value(app: client.Application) -> Optional[str]:
+    from_ = get_from_value(app=app, tenant_id=tenant_id)
```

Comment:
I won't make too much of a fuss if you want to keep it this way, but two things:

1. I would prefer to see "next" instead of "from". But i'll leave that up to you.
2. Can it just be `get_next()` or `get_from()`? 

Same for other functions below, but I'll get to those.